DllImportGenerator/DllImportGenerator/StubCodeContext.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Microsoft.Interop
+{
+    class StubCodeContext
+    {
+        public enum Stage
+        {
+            Setup,
+            Marshal,
+            Pin,
+            Invoke,
```

Comment:
Opinions on Invoke vs Dispatch?

Reply:
I have no real preference - happy to use whatever other folks think makes more sense.

My mind likes to jump from `Dispatch` to `IDispatch` and `Dispatcher` (as in WPF), so I default to `Invoke`, but I'm fine with either.

Reply:
I think this framing makes `Invoke` more attractive. Let's keep it as is.